### ISSUE TITLE
Question about PyTorch training of ResNeSt

### ISSUE THREAD

**Issue by HobbitLong** [zhanghang1989/ResNeSt#65]
Thanks for open-sourcing this nice project, and congratulations!

My question is: for the pre-trained model you released, are the pre-trained with PyTorch, or pre-trained with Gluon and then imported into PyTorch?

I have this question because I saw the [`DropBlock2D`](https://github.com/zhanghang1989/ResNeSt/blob/master/resnest/torch/resnet.py) here is still missing. Then I wonder if it is necessary?

**Comment by zhanghang1989** [comment]
I could not reproduce the results of DropBlock paper, but we found the MXNet implementation can slightly improve the performance. It is okay to train w/o DropBlock.

**Comment by zhanghang1989** [comment]
For training with PyTorch, please go to this page https://hangzhang.org/PyTorch-Encoding/model_zoo/imagenet.html

**Comment by HobbitLong** [comment]
Thanks for your quick reply! I have two further questions:
(1) In MXNet, how large is the difference in accuracy between using or not using DropBlock? I just want to get a quick sense of it
(2) I took a look at the page you pointed to me. I wonder if the `--crop-size` is carelessly misaligned with the paper there, or you just meant it? I saw you use the default 224 for ResNeSt-101 model, 256 for ResNeSt-200 model, and 320 for ResNeSt-269 model, which is smaller than the paper.
